record/menubar/FlashDateSeparators/false.js

```diff
@@ -0,0 +1,55 @@
+const aperture = require('aperture')()
+const delay = require('delay')
+const robot = require('robotjs')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+const { makeAppActive, moveAndResizeApp, compressVideo } = require('../../utils')
+
+module.exports = {
+  run: async (outputPath) => {
+    console.log('> Recording menu bar clock FlashDateSeparators with param set to false')
+    // Set the menu bar menuExtras to only show the clock, it will be on the left of notification center, siri, and spotlight search.
```

Comment:
```suggestion

    // Set the menu bar menuExtras to only show the clock, it will be on the left of notification center, siri, and spotlight search.
```